crates/hdk3/src/lib.rs

```diff
@@ -1,9 +1,482 @@
+//! ## Holochain Development Kit 3.0 (HDK3)
+//!
+//! This is the third major iteration of the holochaind development kit.
+//!
+//! The HDK3 exists to make working with WASM in holochain much easier.
```

Comment:
@maackle oh yeah, as a non-US this isn't anywhere near top of mind for me, the module has been called 'HDK3' forever

i was thinking about it more like tech versions that are more major a difference than semver can convey, like `HTML5`, `CSS3`, `RSS2`, `JSON1`, `MP4`, `USB3`, `5G` or more tangentially like `CAT5` (technically a shortening not an acronym)

this version of HDK is completely different to redux, not even comparable really given the last version was basically a giant macro like `ink!` in polkadot that accentuates the DSL, while this version focuses on making the development experience as close to idiomatic rust as possible by abstracting and downplaying idiosyncrasies

whether or not that justifies a different name... :shrug:  but it's definitely not 'High Definition KKK' :sweat_smile: 

sidenote: a number can also mean the elision of characters, like `a11y`, `i18n`, `a16z`

if you think it's worth changing let's discuss `f2f` because i'm open to the idea but think maybe we could get some more opinions/suggestions in the mix :)

Reply:
> i was thinking about it more like tech versions that are more major a difference than semver can convey, like `HTML5`

That's true, that's another instance of using numbers, but I'd say the vast majority of people's first experience with any kind of HDK is going to be HDK3 and they're not going to have any trouble in conversation with other devs. "By HDK, do you mean HDK 2 or 3?"

Not too partial, yeah maybe we can get other opinions in the mix, this is just mine.

Reply:
@maackle :shrug: i can go either way, but do want to collect feedback, the thing i like is having something more than just semver to convey that this is new and significantly different than what came before it

also, `hdk` is taken on crates.io - https://crates.io/crates/hdk - and bears no resemblance or relationship to this crate

i still see a lot of people calling it "holochain rsm" who have been around a while, and i saw matic rebranded to polygon and got a sweet arbitrary price pump haha

is there something that is neither `hdk` nor `hdk3` that could work?